backend/package.json

```diff
@@ -15,7 +15,7 @@
     "migration:generate": "sequelize migration:generate --config db/config.js --migrations-path db/migrations",
     "prettier": "prettier --write *.js \"**/*.js\"",
     "prettier:check": "prettier -c *.js \"**/*.js\"",
-    "test": "NODE_ENV=test mocha --timeout 10000 --exit --file test/setup"
+    "test": "NODE_ENV=test TEST_TMP_DIR=/tmp/dshop-test-`date '+%Y-%m-%d-%H:%M:%S'` mocha --timeout 10000 --exit --file test/setup"
```

Comment:
Kind of defeats the purpose of using an env var over a const doesn't it?

Reply:
I had to do that in order for models/index.js to use the proper sqlite location. Otherwise it was using the default sqlite URL which is src/backend/db rather that TEST_TMP_DIR
I'm open to any suggestion for simplifying this...

Reply:
If `modes/index.js` and the tests pull the temp dir from the same source(probably somewhere in `models/` that would work.  And you could leave the env var open in case someone wanted to manually set it for whatever reason(using it this way in package.json it can't be overridden without altering package.json).  

This isn't a huge deal, env vars in package.json just irk me a little.

Reply:
Yes that's a better idea. Done. Thank you for the suggestion.